docs/changelog.rst

```diff
@@ -18,6 +18,9 @@ KSQL 5.4.0 includes new features, including:
   Avro record or JSON object, depending on the format in use, or as an anonymous value.
   For more information, refer to :ref:`ksql_single_field_wrapping`.
 
+* A new config `ksql.metrics.tags.custom` for adding custom tags to emitted JMX metrics.
```

Comment:
is it `to emit` or `with emitted`?

Reply:
I have a slight preference for "adding ... to" versus "adding ... with" but I don't feel too strongly.